### PR TITLE
feat: moments section

### PR DESCRIPTION
**Pull request by padmajabhol**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: config/content/Schedule.js [hunk 1]
```diff
@@ -223,10 +223,10 @@ const link4 = ics(event4);
 const link5 = ics(event5);
 
 export default {
-  title: 'Mark Your Calendars!',
-  content:
-    'We have prepared a list of simple steps one needs to follow to become a true hacker. The person who attains all these abilities will yield the power to hack anything.',
-  button: 'https://calendar.google.com/calendar/u/0/r?cid=b0buee3m61snoq0bfi0n5g7hgnof2st4@import.calendar.google.com',
+  title: 'Moments',
+  content: ' Keep tweeting at #hacknitr to get featured',
```
Comment (abhibhaw) [inline config/content/Schedule.js:227]: Hey bhagwan itna jugaad 🤣

File: src/components/Schedule/Schedule.jsx [hunk 2]
```diff
@@ -13,9 +13,9 @@ import { schedule } from '../../../config/content/index';
 function Schedule() {
```
Comment (abhibhaw) [inline src/components/Schedule/Schedule.jsx:13]: again 😆

**Review by abhibhaw — CHANGES_REQUESTED** [review]
Change it from nav menu as well pls

**Review by abhibhaw — APPROVED** [review]
LGTM!! Thanks @padmajabhol  🎉 🎉

**Review by riteshsp2000 — APPROVED** [review]

**Pull request merged**